packages/garbageman.vm/garbageman.vm.nuspec

```
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
+  <metadata>
+    <id>garbageman.vm</id>
+    <version>0.2.4</version>
+    <authors>alphillips-lab</authors>
+    <description>A set of tools designed for .NET heap analysis. These tools offer the following benefits for malware researchers: Ability to extract clear-text; payload (PE Images etc.) from .NET heaps quickly; Easy analysis of encrypted network protocols, signs of data exfiltration, and similar; Ability to overcome malware anti-dumping techniques (psnotify)</description>
```

I would short this description to:
```suggestion
    <description>A set of tools designed for .NET heap analysis.</description>
```


From [our wiki](https://github.com/mandiant/VM-Packages/wiki/Package-Structure#description):
> The description should be short and not include version specific details or other information that is likely to change in a future version.